lib/bolt/core/src/dep/cargo/cli.rs

```diff
@@ -148,8 +148,14 @@ pub async fn build<'a, T: AsRef<str>>(ctx: &ProjectContext, opts: BuildOpts<'a,
 
 						FROM rust:1.72-slim
 
+						RUN apt-get update && apt-get install -y protobuf-compiler pkg-config libssl-dev g++
 						RUN apt-get update && apt-get install -y protobuf-compiler pkg-config libssl-dev g++
 
+						RUN apt-get install --yes libpq-dev wget
+						RUN wget https://github.com/mozilla/sccache/releases/download/v0.2.15/sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz \
+							&& tar xzf sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz \
+							&& mv sccache-v0.2.15-x86_64-unknown-linux-musl/sccache /usr/local/bin/sccache \
+							&& chmod +x /usr/local/bin/sccache
 						RUN apt-get install --yes libpq-dev wget
 						RUN wget https://github.com/mozilla/sccache/releases/download/v0.2.15/sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz \
 							&& tar xzf sccache-v0.2.15-x86_64-unknown-linux-musl.tar.gz \
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 159 to 163]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->double dupe<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/373?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->